mirage/config.js

```diff
@@ -325,6 +325,33 @@ export default function () {
     return this.serialize(builds, 'build');
   });
 
+  this.post('/repo/:repo_id/requests', function (schema, request) {
+    const requestBody = JSON.parse(request.requestBody);
+    const fakeRequestId = 5678;
+    let repository = schema.find('repository', { id: request.params.repo_id });
```

Comment:
Since finding by id is the most common case, `schema.find('repository', request.params.repo_id)` would do this too. But this way is fine too 😀